### DISCUSSION TITLE
Icons not showing up in build

### DISCUSSION THREAD

**Discussion by NikiLentz** [quasarframework/quasar#15683] [CLI - Electron mode] [answered]
Hey, I am creating a small electron app using vue and quasar. 
My main.ts in the renderer part looks like this: 
```
import { createApp } from 'vue'
import './style.css';
import App from './App.vue'



import { Quasar } from 'quasar'

// Import icon libraries
import '@quasar/extras/material-icons/material-icons.css'

// Import Quasar css
import 'quasar/dist/quasar.css'

// Assumes your root component is App.vue
// and placed in same folder as main.js


const myApp = createApp(App)

myApp.use(Quasar, {
  plugins: {}, // import Quasar plugins and add here
})

// Assumes you have a <div id="app"></div> in your index.html
myApp.mount('#app')

```
when running with npm run dev my icons are showing up correctly, but when I build with npm run build using electron-builder I don't see any of the material icons. Where could my config be going wrong?

**Comment by dongwa** [comment, marked as answer]
You need to check `@quasar/extras` is in your dependencies,not in devDependencies.
And why not use`@quasar/cli` to create a Quasar project? You will get Electron out of the box.

**Reply by NikiLentz** [reply]
Yeah I just tried adding Quasar after already creating my app and realizing that vuetify made more problems than it's worth. After posting this I recreated the whole app with the cli and now everything works as expected, so thank you for the tip!
